tests/cases/focus/keyboard_focus.slint

```diff
@@ -0,0 +1,102 @@
+// Copyright © SixtyFPS GmbH <info@slint-ui.com>
```

Comment:
This test is not really testing anythingnew, is it?.  It just will add time to the test without increasing the test coverage.

You should add some ```rust or cpp comments that sends keystroke to it. 
If it is meant to be a manual test only, move it to tests/manual.
otherwise you can also leave it out.

Reply:
I'll move it to manual for now. The navigation itself should be covered in the rust code anyway.